scripts/test/partitionSplitting/system-test-instance.properties

```diff
@@ -65,162 +65,21 @@ sleeper.optional.stacks=SystemTestStack,CompactionStack,IngestStack,PartitionSpl
 # Whether to keep the sleeper table bucket, Dynamo tables related to tables and query results bucket or not.
 sleeper.retain.infra.after.destroy=false
 
-# The maximum number of concurrent compaction tasks to run
-sleeper.compaction.max.concurrent.tasks=300
-
 # Partitions with more than the following number of records in will be split
 # Set to 4 billion so that one round of splitting compactions happen
 sleeper.default.partition.splitting.threshold=20
 
 # How often partition splitting should happen
 sleeper.partition.splitting.period.minutes=10
 
-# The maximum number of files in a partition splitting job
-sleeper.partition.splitting.files.maximum=50
-
-# The frequency with which the garbage collector is run
-sleeper.gc.period.minutes=5
-
-# The size of the batch of files that are ready for garbage collection requested from the State Store
-sleeper.gc.batch.size=2000
-
-# The length of time after a message is consumed from a queue before it becomes visible again
-sleeper.queue.visibility.timeout.seconds=900
-
 # The frequency with which a compaction job sends keep alive messages to the job queue
 sleeper.compaction.keepalive.period.seconds=600
```

Comment:
Maybe we should make sure we test against the default for this?

Reply:
Yes, let's delete this so that the defaults are used.